### ISSUE TITLE
Pest v2.0 support

### ISSUE THREAD

**Issue by pedrogxv** [lukeraymonddowning/pest-plugin-larastrap#2]
Hello, I tried to install `"lukeraymonddowning/pest-plugin-larastrap": "^0.2.0"` with `"pestphp/pest": "^2.0"`, but it gave me the following error:

![image](https://user-images.githubusercontent.com/63373521/227523100-c7d49c05-8518-45cb-87e4-f68e5cd9cd92.png)

I assume it is a problem with the supported version of pest of this package, am I right? And if it it, what about an update regarding it?

my composer.json 'test related packages':
```composer
"lukeraymonddowning/pest-plugin-larastrap": "^0.2.0",
"pestphp/pest": "^2.0",
"pestphp/pest-plugin-laravel": "^2.0",
```

**Comment by qrczak** [comment]
Yeah, i had the same. Would be nice to update this helpful package to support Pest v.2

**Comment by pedrogxv** [comment]
I've just created a PR related to this. Let's see what we can do.

**Comment by lukeraymonddowning** [comment]
Thanks @pedrogxv. Merged and tagged as v0.3.0
